src/main/java/pl/javastart/task/Person.java

```diff
@@ -0,0 +1,65 @@
+package pl.javastart.task;
+
+public class Person {
+    private String firstName;
+    private String lastName;
+    private int age;
+    private String pesel;
+
+    public Person(String firstName, String lastName, int age, String pesel) throws NameUndefinedException, IncorrectAgeException {
+        if (firstName == null) {
+            throw new NameUndefinedException("Imię nie może być null");
+        }
+        if (lastName == null) {
+            throw new NameUndefinedException("Nazwisko nie może być null");
+        }
+        if (firstName.length() < 2) {
+            throw new NameUndefinedException("Imię musi być dłuższe niż dwa znaki");
+        }
+        if (lastName.length() < 2) {
+            throw new NameUndefinedException("Nazwisko musi być dłuższe niż dwa znaki");
+        }
+        if (age < 1) {
+            throw new IncorrectAgeException();
+        }
```

Comment:
Ten warunek sprawdzasz w setterze. Żeby uniknąć powtarzania tej samej walidacji w konstruktorze i setterach to tutaj możesz wywołać `this.setAge(-1)`.
Podobnie w przypadku imienia i nazwiska